### PR TITLE
custom time select caption

### PR DESCRIPTION
**Pull request by alalpod**
Hi, added timeSelectCaption prop to suit other languages.

### CODE REVIEW

File: docs-site/src/examples/show_time.jsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ export default class ShowTime extends React.Component {
             showTimeSelect
             timeFormat="HH:mm"
             timeIntervals={15}
+            timeSelectCaption="Caption"
```
Comment (martijnrusschen) [inline docs-site/src/examples/show_time.jsx]: Maybe say "Time" here?

File: docs-site/src/examples/show_time.jsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ export default class ShowTime extends React.Component {
     <strong>{`    showTimeSelect
     timeFormat="HH:mm"
     timeIntervals={15}
+    timeSelectCaption="Caption"
```
Comment (martijnrusschen) [inline docs-site/src/examples/show_time.jsx]: Maybe say "Time" here?

Comment (alalpod) [reply docs-site/src/examples/show_time.jsx]: "Time" is default prop, so I put a "Caption" here that everyone could see what it does. But if you think there should be "Time" i will change it.

**Comment by martijnrusschen** [comment]
Should be good to go when merge conflicts are resolved.

**Comment by martijnrusschen** [comment]
Closing due inactivity.

**Pull request closed without merging**